model/project_ref.go

```diff
@@ -1910,24 +1911,25 @@ func (p *ProjectRef) MakeUnrestricted() error {
 	return nil
 }
 
-func (p *ProjectRef) UpdateAdminRoles(toAdd, toRemove []string) error {
```

Comment:
Why is this being done for projectref only and not reporef?

Reply:
If you take a look at repo ref, we don't have to look up what the role is because we already know the structure of it, so it's not possible to have an error before we get to the loop which handles it more dynamically.